client/stylesheets/_toCat.scss

```diff
@@ -15,10 +15,16 @@ GLOBAL | GLOBAL DIVS | HEADER | FOOTER **/
     background-color: yellow;
 }
 
+.tooltip > .tooltip-inner {
+    background-color: $color-hashtag !important;
+    height: 30% !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces